### PR TITLE
Portage utils

### PR DESCRIPTION
**Pull request by SoapGentoo**
`ebuild portage-utils-0.94.2.ebuild clean test install`:

    $ cat /var/tmp/portage/app-portage/portage-utils-0.94.2/build-info/NEEDED
    /usr/bin/q libcrypto.so.3,libb2.so.1,libgomp.so.1,libc.so.6

`USE="static" ebuild portage-utils-0.94.2.ebuild clean test install`:

    $ cat /var/tmp/portage/app-portage/portage-utils-0.94.2/build-info/NEEDED
    /usr/bin/q

### CODE REVIEW

File: app-portage/portage-utils/portage-utils-9999.ebuild [hunk 1]
```diff
@@ -18,50 +16,60 @@ else
...
+	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
+}
+
+pkg_setup() {
+	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
+}
+
+src_prepare() {
+	default
+	[[ ${PV} == *9999 ]] && eautoreconf
```
Comment (SoapGentoo) [inline app-portage/portage-utils/portage-utils-9999.ebuild:63]: @grobian I've added this. In general, if someone uses the live ebuild, we don't want to necessarily rely on the fact that the autotools stuff was regenerated after every merge.

**Review by grobian — APPROVED** [review]
lgtm thanks!

File: app-portage/portage-utils/portage-utils-9999.ebuild (hunk 1 above)
Comment (grobian) [reply app-portage/portage-utils/portage-utils-9999.ebuild:63]: I'm fine with that, don't think it holds for all scenarios, but the actions would catch if something doesn't work.  I don't use the 9999, might as well drop it, not sure how useful it is.

Comment (SoapGentoo) [reply app-portage/portage-utils/portage-utils-9999.ebuild:63]: tell me what you want - I can keep it or remove it

Comment (grobian) [reply app-portage/portage-utils/portage-utils-9999.ebuild:63]: keep it, it seems like a nice safeguard

**Pull request closed without merging**